server/core/src/test/java/org/infinispan/server/core/MockProtocolServer.java

```diff
@@ -0,0 +1,51 @@
+package org.infinispan.server.core;
+
+import io.netty.channel.Channel;
+import io.netty.channel.ChannelInboundHandler;
+import io.netty.channel.ChannelInitializer;
+import io.netty.channel.ChannelOutboundHandler;
+import io.netty.channel.group.ChannelMatcher;
+import org.infinispan.server.core.configuration.MockServerConfigurationBuilder;
+import org.infinispan.server.core.configuration.ProtocolServerConfiguration;
+import org.infinispan.server.core.transport.NettyTransport;
+
+public class MockProtocolServer extends AbstractProtocolServer {
+
+   public MockProtocolServer(String protocolName, NettyTransport transport) {
+      super(protocolName);
+      configuration = new MockServerConfigurationBuilder()
+            .defaultCacheName("dummyCache")
```

Comment:
I feel like the default cache name and the port should be static final class field so that a test can more easily verify them.